<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projet BLE &amp; Sensort : Station Météo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabTextData">
       <attribute name="title">
        <string>Données textuelles</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBoxTemperature">
          <property name="title">
           <string>Température</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="labelTempValue">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>--.-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTempUnit">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>°C</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxHumidity">
          <property name="title">
           <string>Humidité</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="labelHumidityValue">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>--.-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelHumidityUnit">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxPressure">
          <property name="title">
           <string>Pression</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="labelPressureValue">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>----.--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPressureUnit">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>hPa</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGraphs">
       <attribute name="title">
        <string>Graphiques</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCustomPlot" name="widgetTempPlot" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCustomPlot" name="widgetHumidityPlot" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCustomPlot" name="widgetPressurePlot" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabConfig">
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBoxConnection">
          <property name="title">
           <string>Mode de connexion</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QRadioButton" name="radioButtonSerial">
             <property name="text">
              <string>Port série</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="labelSerialPort">
               <property name="text">
                <string>Port:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxSerialPorts"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonRefreshPorts">
               <property name="text">
                <string>Rafraîchir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonBLE">
             <property name="text">
              <string>Bluetooth LE</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="pushButtonScanBLE">
               <property name="text">
                <string>Rechercher appareils BLE</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="listWidgetBLEDevices"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pushButtonConnect">
            <property name="text">
             <string>Connecter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDisconnect">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Déconnecter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelStatus">
          <property name="text">
           <string>Statut: Non connecté</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionExportData"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExportData">
   <property name="text">
    <string>Exporter les données...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>À propos</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
